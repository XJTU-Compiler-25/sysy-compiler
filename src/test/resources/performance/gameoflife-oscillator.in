version https://git-lfs.github.com/spec/v1
oid sha256:9cdd495224f5fa3eea564fcbc3f1bd3987cb2afe33227e806054dfc0a07142f7
size 2561
