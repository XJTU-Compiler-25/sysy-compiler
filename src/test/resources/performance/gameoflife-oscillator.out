version https://git-lfs.github.com/spec/v1
oid sha256:6e4f5cb0f53239110bf69d29f1b251300bfedb91dabacb2830745c6719693a24
size 2552
